src/validations/class.js

```diff
@@ -12,4 +12,37 @@ const validateId = (req, res, next) => {
   });
 };
 
-module.exports = { validateId };
+const validateUpdate = (req, res, next) => {
+  const classValidation = Joi.object({
+    hour: Joi.string()
+      .pattern(/^([01]?[0-9]|2[0-3]):([0-5][0-9])$/)
+      .messages({
+        'string.pattern.base':
+        'The field must be a valid hour(for example 09:30 )',
+      }),
+    day: Joi.string()
```

Comment:
Day must be an enum with the days of the week.
You also have to change the message and the Class schedule